common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -66,6 +66,13 @@ class SequenceFields(object):
         scope=Scope.settings,
     )
 
+    self_paced = Boolean(
+        display_name=_("Course is self-paced"),
+        help=_("ask Eric to remove this field"),
+        default=False,
+        scope=Scope.settings,
+    )
```

Comment:
DEBUG: I had to add this field to be able to inspect the inherited `self_paced` value in `seq_module.py`